templates/_layout.html

```diff
@@ -27,6 +27,7 @@
 
     <link rel="icon" type="image/png" href="https://assets.ubuntu.com/v1/fdc99abe-ico_16px.png" sizes="16x16" />
     <link rel="icon" type="image/png" href="https://assets.ubuntu.com/v1/0f3c662c-ico_32px.png" sizes="32x32" />
+    <link rel="apple-touch-icon" type="image/png" href="https://assets.ubuntu.com/v1/565d2563-apple-touch.png" sizes="180x180" />
```

Comment:
Doesn't matter much for now, as it is ready, but ideally this should be fixed separately as it is totally unrelated to FSF changes.

Reply:
Okay I'll make sure that doesn't happen later on but it was so small ...

Reply:
I'll make sure it doesn't happen next time!